questions.md

```diff
@@ -0,0 +1,9 @@
+#Questions:
+
+1. If i want to make a pull request to repo of other user, do i need to clone it first?
```

Comment:
You do not need to clone to make a pull request, you can submit a pull request by selecting "create a new branch for this commit and start a pull request" right in GitHub's online editor. However, you can only do this in repositories  that you contribute to. If you are not a collaborator, you will have to make a [fork](https://help.github.com/articles/fork-a-repo/) and [make a pull request from your fork to the original repo](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).